doc/mgr/modules.rst

```diff
@@ -74,9 +78,11 @@ level is mapped to the module python logging level as follows:
 * <= +inf is DEBUG
 
 We can unset the module log level and fallback to the mgr daemon logging level
-by running the following command::
+by running the following command:
+
+.. prompt:: bash #
 
-  ceph config set mgr mgr/<module_name>/log_level ''
+   ceph config set mgr mgr/<module_name>/log_level ''
 
 By default, modules' logging messages are processed by the Ceph logging layer
```

Comment:
I just wanted to say how nice it is to see the plural possessive used correctly in technical documentation. Here's where this line came into the docs, and here is where credit is due:

Author: Ricardo Dias <rdias@suse.com>
Date:   Wed Oct 16 11:31:40 2019 +0100

    doc/mgr: update mgr modules' logging instructions
    
    Signed-off-by: Ricardo Dias <rdias@suse.com>

Reply:
...and then, on the very next line, the predicative "it's" is used where the possessive "its" should have been.